frontend/coreproject/src/lib/components/pages/anime_info/Index.svelte

```diff
@@ -1 +1,24 @@
-<p>Hello</p>
+<script lang="ts">
+  import { page } from '$app/stores';
+
+	import { anime_list } from '$data/mock/anime_list.ts';
+	import AnimeMainDetails from './AnimeMainDetails.svelte';
+
+  let anime_id = Number($page.params.id);
+
+	let anime = anime_list?.find((anime: any) => anime.id === anime_id);
+</script>
+
+<div class="anime_info overflow-hidden rounded-tl-3xl">
+	<div
+		class="anime_basic_details relative h-screen bg-cover"
+		style="background-image: url({anime?.bannerBackgroundImage ?? ''});"
+	>
+		<div
+			class="gradient absolute h-full w-full bg-gradient-to-t from-surface-500 to-surface-500/50"
+		/>
+		<div class="absolute h-full w-full md:p-16 2xl:p-28">
+			<AnimeMainDetails {anime} />
+		</div>
+	</div>
```

Comment:
![image](https://user-images.githubusercontent.com/61817579/230700147-c73a9bc0-dd23-446f-a4c5-c3ce900b4cfe.png)


See this too? Why is the top nav sticky?

Reply:
Yea I noticed it too
I think its some skeleton things for header

I will take a look again ^_^

Reply:
>  I think its some skeleton things for header

Which is true. Open an discussion on their github repo.